frontend/src/customer/specification/Specification.js

```diff
@@ -0,0 +1,196 @@
+import React, {Component} from 'react';
+import './Specification.scss'
+import StarRatings from "react-star-ratings";
+
+class Specification extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            source: "https://www.swiatobrazu.pl/zdjecie/artykuly/550890/wiewiorka.png",
+            alt: "photo",
+            average: 5.0,
+            star: 1.0,
+            name: "Nazwa salonu",
+            street: "Owaka 4",
+            city: "Wrocław",
+            person: "Michałek",
+            servicename: "Fryzjer damski",
+            serviceprice: 60
+        };
+    }
+
+    /* Sprawdza czy gwiazdka jest biała czy czerwona */
+    CheckStar(){
```

Comment:
Nazwy funkcji raczej małą literą. Jeśli potrzebny jest komentarz wyjaśniający co robi funkcja to znaczy, że źle jest nazwana.